frontend/components/Signer.tsx

```
@@ -0,0 +1,965 @@
+          return; // Skip this expected warning
+        }
+        originalWarn(message, ...args);
+      };
+      console.warn = warnOverride;
+
+      try {
+        // Use igloo-core's cleanup - it handles the manual cleanup internally
+        // cleanupBifrostNode(nodeRef.current);
+      } catch (error) {
+        console.error('Unexpected error during cleanup:', error);
+      } finally {
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)